src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethod_1379666.java

```diff
@@ -5,36 +5,23 @@
  */
 public class InputJavadocMethod_1379666 {
     /**
-     * @throws BadStringFormat some text
+     * @throws Exception some text
      */
-    public void ok() throws BadStringFormat {
+    public void ok() throws Exception {
     }
 
 
     /**
      * Some comment.
-     * @throws com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod.InputJavadocMethod_1379666.BadStringFormat some text
```

Comment:
this input file checks the same exception in different `throws` formats:

- BadStringFormat
- com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod.InputJavadocMethod_1379666.BadStringFormat
- InputJavadocMethod_1379666.BadStringFormat

You should not make them all one format - `Exception`.
Please replace only `com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod.InputJavadocMethod_1379666.BadStringFormat` with `java.lang.Exception` in `error1()` and please restore everything else in this file.

Reply:
Alright 👍

Reply:
@baratali Done. Please check.

Reply:
@baratali ping.

Reply:
@Gaurav-Punjabi , hey, sorry for delay. You should replace it with `java.lang.Exception`, not just `Exception` (mind the package name)

Reply:
Done

Reply:
Done